wscript

```diff
@@ -102,6 +102,42 @@ REFDLLS = [
 	RefDll('gles3compat', False),
 ]
 
+def process_extra_projects_opts(ctx):
```

Comment:
IMO extra projects thing should migrate to waifulib/subproject.py after a while...